aws/inputseeders/aws/masterdata.json

```diff
@@ -648,6 +693,23 @@
         "/var/log/ecs/audit.log": {
             "FilePath": "/var/log/ecs/audit.log",
             "TimeFormat": "%b %d %H:%M:%S"
+        },
+        "/ec2/EC2Launch.log": {
+            "FilePath": "/ec2/EC2Launch.log",
+            "TimeFormat": "%Y/%m/%d %H:%M:%S%Z:"
+        },
+        "/ec2/UserdataExecution.log": {
+            "FilePath": "/ec2/UserdataExecution.log",
+            "TimeFormat": "%Y/%m/%d %H:%M:%S%Z:"
+        },
+        "/cwa/amazon-cloudwatch-agent.log": {
+            "FilePath": "/cwa/amazon-cloudwatch-agent.log",
+            "TimeFormat": "%H: %M: %S%y%b%-d"
+        },
+        "/awsssm/amazon-ssm-agent.log": {
+            "FilePath": "/awsssm/amazon-ssm-agent.log",
+            "TimeFormat": "%Y-%m-%d %H:%M:%S",
+            "MultiLinePattern": "^INFO"
```

Comment:
Are these the windows logs? Lest make them Windows formatted instead of something hard to follow.

Reply:
Yes and no. The FilePath is used to identify the windows log and also the logstream name, hence using a function to translate between them. Would it be better to create a second parameter?

Reply:
Might be better to include the full path. I.e C:/ etc 
Otherwise this can't be customised by people. What would happen if I want to log something on the D: Drive in this case? 

Looking at https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-logs-logstream.html 
`:` is not allowed as part of the name so we could just do a replace when setting the logstream to remove that 


C/ProgramFiles/ is pretty understandable

Reply:
Cool, done

Reply:
not quite what I was thinking 

For the FilePath It should be the real OS path. We just do the removal of the : when the logstream property is set
Then the user understands and enters a real path and we handle the weirdness of AWS underneath. With the configuration as it is we are now showing the weirdness to users and making them have to understand it

Reply:
Cool, done